bootstrap_salt/fab_tasks.py

```diff
@@ -32,6 +32,21 @@ def aws(x):
 
 
 @task
+def environment(x):
+    env.environment = str(x).lower()
+
+
+@task
+def application(x):
+    env.application = str(x).lower()
+
+
+@task
+def config(x):
+    env.config = str(x).lower()
```

Comment:
Without the environment flag, https://github.com/ministryofjustice/bootstrap-salt/blob/bee3703fc4b484c50b4be830d28e50e96f4d514d/bootstrap_salt/fab_tasks.py#L41 complains:
```
oskar@moj-deploy:~/moj/bootstrap-salt$ fab application:courtfinder aws:dev config:/path/to/courtfinder-dev.yaml setup

[ERROR] Please specify an environment, e.g 'environment:dev'
oskar@moj-deploy:~/moj/bootstrap-salt$
```

With the environment task, but no config parameter:
```
oskar@moj-deploy:~/moj/bootstrap-salt$ fab application:courtfinder aws:dev environment:development  setup

[ERROR] Please specify a config file, e.g 'config:/tmp/sample-application.yaml'
```